aspnet/fundamentals/startup.rst

```diff
@@ -52,7 +52,7 @@ Just as with ``Configure``, it is recommended that features that require substan
   :dedent: 8
   :emphasize-lines: 4,7,11
 
-Adding services to the services container makes them available within your application via :doc:`dependency injection <dependency-injection>`. Just as the ``Startup`` class is able to specify dependencies its methods require as parameters, rather than hard-coding to a specific implementation, so too can your middleware, MVC controllers and other classes in your application.
+Adding services to the services container makes them available within your application via :doc:`dependency injection <dependency-injection>`. The methods of the ``Startup`` class require particular dependencies as parameters. This way, the ``Startup`` class can specifiy them dynamically, rather than hardcoding these to a specific implementation. The same goes for all the following layers of your application: the middleware, MVC controllers and other classes. They all receive their dependencies from the services container.
```

Comment:
Can you just delete everything after the first sentence? 

Adding services to the services container makes them available within your application via :doc:`dependency injection <dependency-injection>`. ~~Just as the ``Startup`` class is able to specify dependencies its methods require as parameters, rather than hard-coding to a specific implementation, so too can your middleware, MVC controllers and other classes in your application.~~